keps/NNNN-kep-template/kep.yaml

```diff
@@ -0,0 +1,23 @@
+title: KEP Template
+authors:
+  - "@janedoe"
+owning-sig: sig-xyz
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
```

Comment:
Maybe out of scope for this change, but can we just rely on git history instead of tracking these dates here? I think last-updated is rarely accurate anyway.

Reply:
I considered that, but didn't want to argue it :)  I'd be ok to make that change too

Reply:
dropped in next push